library/lib_pid.ks

```diff
@@ -1,5 +1,8 @@
 // This file is distributed under the terms of the MIT license, (c) the KSLib team
 
+HUDTEXT("This library file has been superseded by the kOS inbuilt PIDloop() function.", 30, 2, 30, RED, FALSE).
+HUDTEXT("It is maintained for example purposes only, please see the kOS documentation for more.", 30, 2, 30, RED, FALSE).
+
```

Comment:
If your goal is to have this appear on screen for the user when a program tries to `run lib_pid.`, so they are aware of the situation, then keep in mind that they won't see any indication in this on-screen message that the phrase "This library file" is talking about lib_pid.  It could be coming from any one of a number of things they're running.

I'd suggest editing "this library file has been superceeded..." and changing it to "lib_pid.ks has been superceeded..."